src/pages/uxp/reference-js/Global Members/HTML DOM/Element.md

```diff
@@ -171,6 +175,21 @@ The width of the element
 ## innerHTML : `string`
 
 
+<a name="element-innerhtml" id="element-innerhtml"></a>
+
+## innerHTML
+**See**
+
+- https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML
+- https://w3c.github.io/DOM-Parsing/#the-innerhtml-mixin
+
+
+| Param | Type |
+| --- | --- |
+| value | `string` | 
+
+
+
```

Comment:
Either it is a property (i.e., `innerHTML : string`, in which case it doesn't have paramters, or it should be (which I don't think is the case) `innerHTML(value)`, in which case the params table "can stay"

Reply:
Addressed manually.